app/javascript/mastodon/reducers/accounts_map.js

```diff
@@ -7,6 +8,8 @@ const initialState = ImmutableMap();
 
 export default function accountsMap(state = initialState, action) {
   switch(action.type) {
+  case ACCOUNT_LOOKUP_FAIL:
```

Comment:
Should we also cater here for `ACCOUNT_FETCH_FAIL`?

Reply:
No, `ACCOUNT_FETCH_FAIL` is for getting an `Account` object from its `id`, `accounts_map` is for storing a mapping from `acct` handle to `id`.